lib/signalfx/tracing.rb

```diff
@@ -25,7 +27,12 @@ def configure(tracer: nil,
 
           if auto_instrument
             Register.available_libs.each_pair do |key, value|
-              value.instrument
+              begin
+                value.instrument
+              rescue Exception => e
+                @logger.error { e.inspect }
```

Comment:
not that I understand ruby but is `inspect` clear enough to the user? I mean, don't we need to add something like "instrumentation blag cannot be installed: <error_message>"?